tests/es6modules/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -80,7 +80,7 @@ export default async function foo() {}
 exports[`export_default_function_declaration_async.js - flow-verify 1`] = `
 export default async function foo() {}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-export default (async function foo() {});
+export default async function foo() {}
```

Comment:
See https://github.com/prettier/prettier/pull/5303#discussion_r228407664

Reply:
resolved in #5303